types/mocha/index.d.ts

```diff
@@ -180,6 +180,13 @@ declare class Mocha {
      */
     delay(): boolean;
 
+    /**
+     * Fails test run if no tests encountered with exit-code 1.
+     *
+     * @see https://mochajs.org/api/mocha#failZero
+     */
+    failZero(): boolean;
```

Comment:
According to https://mochajs.org/api/mocha#failZero, this function should take an optional boolean parameter, no? And the return type should be `this`:

```suggestion
    failZero(enabled?: boolean): this;
```

Reply:
It's up to you, you don't have to fix the existing ones if you don't want, but that would be nice.

Reply:
Only fixed the current method for now, I'll see if I've time to change the others in a follow up PR.